lib/Command/InstallDefaultFonts.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace OCA\Richdocuments\Command;
+
+use OCA\Richdocuments\Service\FontService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class InstallDefaultFonts extends Command {
+	private FontService $fontService;
+
+	public function __construct(FontService $fontService) {
+		parent::__construct();
+		$this->fontService = $fontService;
+	}
```

Comment:
We can make use of PHP 8 constructor property promotion:

https://php.watch/versions/8.0/constructor-property-promotion

```suggestion
	public function __construct(private FontService $fontService) {
		parent::__construct();
	}
```